parsl/configs/bridges.py

```diff
@@ -0,0 +1,36 @@
+from parsl.config import Config
+from parsl.providers import SlurmProvider
+from parsl.launchers import SrunLauncher
+from parsl.executors import HighThroughputExecutor
+
+""" This config assumes that it is used to launch parsl tasks from the login nodes
+of Bridges at PSC. Each job submitted to the scheduler will request 2 nodes for 10 minutes.
+"""
+
+config = Config(
+    executors=[
+        HighThroughputExecutor(
+            label='Bridges_HTEX_multinode',
+            max_workers=1,
+            provider=SlurmProvider(
+                'YOUR_PARTITION_NAME',  # Specify Partition / QOS, for eg. RM-small
```

Comment:
this shouldn't have "partition=" before it?

Reply:
It's a positional arg, so not necessary.

Reply:
it seems like it would be cleaner/clearer to have it

Reply:
but I'll approve anyhow